src/Ajax.php

```diff
@@ -321,10 +321,18 @@ public static function createTabs(
             $html_tabs = "";
             $html_sele = "";
             $i = 0;
+
+            // Hide tabs on item creation
+            $display_class = "";
+            if (is_a($type, CommonDBTM::class, true) && $type::isNewId($ID)) {
+                $display_class = "d-none";
```

Comment:
Maybe it should be hidden only if there is only one tab (not sure if it is possible to have multiples tabs on item creation).

Reply:
Can't hurt to check if there is only one tab, I'll add this

Reply:
creation from an asset template (ex computer) may be considered as isNewId = true but have complete set of tabs